src/Infrastructure/Multitenancy/TenantManager.cs

```diff
@@ -62,10 +62,27 @@ public async Task<Result<TenantDto>> GetByIssuerAsync(string issuer)
         return await Result<TenantDto>.SuccessAsync(tenantDto);
     }
 
+    private string GetSecureConnectionString(string? connectionString){
+        string result = string.Empty;
+        if(!string.IsNullOrWhiteSpace(connectionString)){
+            var stringParts = connectionString.Split(new char[]{';'});
+            var newConnectionString = stringParts
+                .Where(s=>s.Contains("password", StringComparison.InvariantCultureIgnoreCase)==false)//password=....s
+                .Where(s=>s.Contains("User ID", StringComparison.InvariantCultureIgnoreCase)==false)//User ID=sa
```

Comment:
Won't this be not valid for other DB providers? where they will not use `User ID` ?

Reply:
> Won't this be not valid for other DB providers? where they will not use `User ID` ?

you are right anyway it was a way to hide those 2 properties if any.
....
How do you manage to do PR (draft)?

Reply:
![image](https://user-images.githubusercontent.com/27079402/148083949-7383d2e5-2f40-435b-bce3-5532d9d18b46.png)
analyzing... the only provider among those discussed is mysql that has more ways to write the user parameter ....

Reply:
i made it in draft mode. I believe there should be a better way to do this, rather that processing every connection string.